src/midi/midi_fluidsynth.cpp

```diff
@@ -178,6 +179,12 @@ static std::string find_sf_file(const std::string &name)
 	return "";
 }
 
+MidiHandlerFluidsynth::MidiHandlerFluidsynth()
+        : soft_limiter("FSYNTH", prescale_level, expected_max_frames)
+{
+	stream.resize(expected_max_frames * 2); // L & R channels
```

Comment:
This is suspicious.

Reply:
(Same reason as the one-time resize in the SoftLimiter)

Reply:
I commented on this in new round of review - this means this vector's constructor should be called with `size_t` parameter in this class initializer list.

Reply:
Absolutely; thanks @dreamer.
There were some stack-exchange answers that discourage using vector's constructor to do most operations (default values, default size, etc..) and favour the explicit calls just for readability.  But I actually prefer moving this into the initializer list, and the default-size is on-par with std::array's default size template arg too.